static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-mandatory-long.js

```diff
@@ -1,7 +1,7 @@
-export const signInGateMandatoryLongTestRunUk = {
-	id: 'SignInGateMandatoryLongTestRunUk',
-	start: '2022-09-20',
-	expiry: '2022-10-01',
+export const signInGateMandatoryLongBucketingTestRun = {
+	id: 'SignInGateMandatoryLongBucketingTestRun',
+	start: '2022-10-05',
```

Comment:
Would you like to change the date perhaps? I don't think it's important since you're in the timeframe you want - just thought I mention this.

Reply:
No need to change the date - all good